src/commands/createStaticWebApp/StaticWebAppNameStep.ts

```diff
@@ -62,14 +66,11 @@ export class StaticWebAppNameStep extends AzureNameStep<IStaticWebAppWizardConte
     }
 
     private async isSwaNameAvailable(context: IStaticWebAppWizardContext, resourceName: string): Promise<boolean> {
-        // Static Web app names must be unique to the current resource group.
-        try {
-            await context.client.staticSites.getStaticSite(resourceName, resourceName);
-            return false;
+        // Our implementation is that Static Web app names must be unique to the subscription.
+        listOfSites = listOfSites || await uiUtils.listAll(context.client.staticSites, context.client.staticSites.list());
```

Comment:
It seems possible that a recently created SWA's name will show as available if we use the cached `listOfSites` instead of always getting a fresh list

Reply:
Also the cached list won't be accurate if they select a different subscription